bin/compile

```diff
@@ -8,11 +8,21 @@ echo "-----> Install ImageMagick"
 
 BUILD_DIR=$1
 CACHE_DIR=$2
+ENV_DIR=$3
 VENDOR_DIR="$BUILD_DIR/vendor"
 mkdir -p $VENDOR_DIR
 INSTALL_DIR="$VENDOR_DIR/imagemagick"
 mkdir -p $INSTALL_DIR
-IMAGE_MAGICK_VERSION="${IMAGE_MAGICK_VERSION:-6.9.5-10}"
+if [ -d $ENV_DIR ]; then
```

Comment:
I did run it on a live app, and it worked to install the right version of ImageMagick. But the app failed because ImageMagick requires some version-specific configuration setup that I couldn't figure out how to add via the buildback. So I ended up using a different approach (Linux 'file' command instead of 'identify') and abandoned this direction.